pkg/native/serializers/serializer_cdx.go

```diff
@@ -47,7 +47,9 @@ func (s *CDX) Serialize(bom *sbom.Document, _ *native.SerializeOptions, _ interf
 	doc.SerialNumber = bom.Metadata.Id
 	ver, err := strconv.Atoi(bom.Metadata.Version)
 	// TODO(deprecation): If version does not parse to int, there's data loss here.
-	if err == nil {
+	if err != nil || ver < 1 {
+		doc.Version = 1 // Default to 1
+	} else {
```

Comment:
Since the CDX serializer is the only user of the version field, change the init of the version number from 0 to 1 here:

https://github.com/protobom/protobom/blob/5612e368b95d757401f322ed4939b3957d8d40a0/pkg/sbom/document.go#L26

If we get a zero here it's fine to bump it to 1